sw/device/lib/dif/autogen/dif_lc_ctrl_autogen.inc

```diff
@@ -0,0 +1,42 @@
+// Copyright lowRISC contributors.
```

Comment:
Is there a reason to name these files `.inc`?  Bazel really doesn't like public headers called anything except `.h`.

To me, any header file named anything apart from `.h` usually means the file can't be fed to the compiler alone (ie: it's incomplete as an compiler input and is usually involved in some macro and include trickery ([example](https://chromium.googlesource.com/chromiumos/platform/ec/+/master/board/eve/gpio.inc))).

These seem to be complete headers (ifdef guards, include what they use), so IMHO, they should be named `.h`.  It does appear that they aren't necessarily meant to be _public_ headers (ie: they get included by their DIF's main header file and nothing else.  There are some schools of though that allow names like `.inc` for non-public headers.  I don't happen to adhere to those schools of thought).

Would it be acceptable to name these as `.h` files?